assets/js/components/settings/SettingsCardKeyMetrics.js

```diff
@@ -69,43 +73,71 @@ export default function SettingsCardKeyMetrics() {
 	return (
 		<Layout title={ __( 'Key Metrics', 'google-site-kit' ) } header rounded>
 			<div className="googlesitekit-settings-module googlesitekit-settings-module--active googlesitekit-settings-user-input">
-				<SettingsKeyMetrics />
+				{ isUserInputCompleted && (
+					<Fragment>
+						<SettingsKeyMetrics loading={ loading } />
+
+						<Grid>
+							<Row>
+								<Cell size={ 12 }>
+									<UserInputPreview
+										settingsView
+										loading={ loading }
+									/>
+								</Cell>
+							</Row>
+						</Grid>
+					</Fragment>
+				) }
 
-				<Grid>
-					{ isUserInputCompleted && (
-						<Row>
-							<Cell size={ 12 }>
-								<UserInputPreview settingsView />
-							</Cell>
-						</Row>
-					) }
+				{ isUserInputCompleted === false && (
+					<Fragment>
+						<SettingsKeyMetrics />
 
-					{ isUserInputCompleted === false && (
-						<Row>
-							<Cell
-								className="googlesitekit-user-input__notification"
-								size={ 12 }
-							>
-								<p>
-									<span>
-										{ __(
-											'Answer 3 quick questions to help us show the most relevant data for your site',
-											'google-site-kit'
-										) }
-									</span>
-								</p>
+						<Grid>
+							<Row>
+								<Cell
+									className="googlesitekit-user-input__notification googlesitekit-overflow-hidden"
```

Comment:
The use of `overflow: hidden` ensures the fixed width preview blocks don't overflow the page at narrower viewport widths.